utils/sniff/sniffer.go

```diff
@@ -145,3 +152,35 @@ func (sniffer *Sniffer) Validate(ctx context.Context, a *check.Action) {
 		a.Infof("Capture executed on %s (%s): %s", sniffer.target.String(), sniffer.target.NodeName(), strings.Join(sniffer.cmd, " "))
 	}
 }
+
+func (sniffer *Sniffer) stopAndWait(ctx context.Context) error {
+	select {
+	case err := <-sniffer.exited:
+		// When running in assert mode, it is an error if tcpdump already stopped
+		// at this point, as we may have missed packets. That's not the case in
+		// sanity mode, as configured to stop after receiving a single packet.
+		if err == nil && sniffer.mode == ModeAssert {
+			return errors.New("tcpdump terminated unexpectedly")
+		}
+
+		return err
+
+	default:
+		wctx, wcancel := context.WithTimeout(ctx, time.Second)
```

Comment:
Does "w" stand for "wait"?

Reply:
Yep, that was the intention. I've just dropped it and aliased `ctx` though now, as it isn't used otherwise, and looks clearer.